applications/DEMApplication/python_scripts/DEM_benchmarks_class.py

```diff
@@ -4203,11 +4213,828 @@ def compute_errors(self, velocity_list_outfile_name):  #FINALIZATION STEP
+                    displacement_node = sqrt(displacement[0]*displacement[0] + displacement[1]*displacement[1] + displacement[2]*displacement[2])
+
+                    i=0
+                    data  = open("benchmark" + str(sys.argv[1]) + "_graph%s.dat" % i, 'a')
+                    data.write(str("%.8g"%time).rjust(12)+" "+str("%.6g"%force_node).rjust(13)+" "+str("%.6g"%angular_node).rjust(13)+" "+str("%.6g"%displacement_node).rjust(13)+"\n")
+                    data.flush()
+
+
+            for node in modelpart.Nodes:
+                if node.Id == 42:           ### stage 1
+
+                    force_node_x = node.GetSolutionStepValue(TOTAL_FORCES)[0]
+                    force_node_y = node.GetSolutionStepValue(TOTAL_FORCES)[1]
+                    force_node_z = node.GetSolutionStepValue(TOTAL_FORCES)[2]
+
+                    angular_node_x = node.GetSolutionStepValue(ANGULAR_VELOCITY)[0]
+                    angular_node_y = node.GetSolutionStepValue(ANGULAR_VELOCITY)[1]
+                    angular_node_z = node.GetSolutionStepValue(ANGULAR_VELOCITY)[2]
+
+                    displacement = [0]*3
+                    displacement[0] = node.X - node.X0
+                    displacement[1] = node.Y - node.Y0
+                    displacement[2] = node.Z - node.Z0
+
+                    force_node = sqrt(force_node_x*force_node_x + force_node_y*force_node_y + force_node_z*force_node_z)
+                    angular_node = sqrt(angular_node_x*angular_node_x + angular_node_y*angular_node_y + angular_node_z*angular_node_z)
+                    displacement_node = sqrt(displacement[0]*displacement[0] + displacement[1]*displacement[1] + displacement[2]*displacement[2])
+
+                    i=1
+                    data  = open("benchmark" + str(sys.argv[1]) + "_graph%s.dat" % i, 'a')
+                    data.write(str("%.8g"%time).rjust(12)+" "+str("%.6g"%force_node).rjust(13)+" "+str("%.6g"%angular_node).rjust(13)+" "+str("%.6g"%displacement_node).rjust(13)+"\n")
+                    data.flush()
+
+            for node in modelpart.Nodes:
+                if node.Id == 71:           ### stage 2
+
+                    force_node_x = node.GetSolutionStepValue(TOTAL_FORCES)[0]
+                    force_node_y = node.GetSolutionStepValue(TOTAL_FORCES)[1]
+                    force_node_z = node.GetSolutionStepValue(TOTAL_FORCES)[2]
+
+                    angular_node_x = node.GetSolutionStepValue(ANGULAR_VELOCITY)[0]
+                    angular_node_y = node.GetSolutionStepValue(ANGULAR_VELOCITY)[1]
+                    angular_node_z = node.GetSolutionStepValue(ANGULAR_VELOCITY)[2]
+
+                    displacement = [0]*3
+                    displacement[0] = node.X - node.X0
+                    displacement[1] = node.Y - node.Y0
+                    displacement[2] = node.Z - node.Z0
+
+                    force_node = sqrt(force_node_x*force_node_x + force_node_y*force_node_y + force_node_z*force_node_z)
```

Comment:
`force_node = sqrt(sum([node.GetSolutionStepValue(TOTAL_FORCES)[i]**2 for i in range(2)]))` and there is no need to define force_node_x, etc.